test/stream-write-read-perf.c

```diff
@@ -42,14 +43,15 @@ on_read (udx_stream_t *handle, ssize_t read_len, const uv_buf_t *buf) {
   stats.last_read_ms = uv_hrtime() / 1000000;
 
   if (stats.bytes_read == options.size_bytes) {
-    udx_stream_destroy(handle);
+    // udx_stream_destroy(handle);
```

Comment:
Leftover?

Reply:
moved the stream destroy (for both streams) to the on_ack event.